.travis.yml

```diff
@@ -108,3 +104,12 @@ matrix:
         - memcached
         - redis-server
         - postgresql
+notifications:
+  webhooks:
+    urls:
+      - https://zeus.ci/hooks/fa079cf6-8e6b-11e7-9155-0a580a28081c/public/provider/travis/webhook
```

Comment:
Going to assume this is fine.

Reply:
Yeah its safe. Travis signs hooks and we still verify signature, and this gives no other access.